### PR TITLE
Add pre-commit hook

### PR DESCRIPTION
**Pull request by trevordmiller**
# Changes

- Fix some compile errors
- Add pre-commit hook to run `yarn verify`, same as CI
- Version bump for new release on npm

# Result For User

Commits take a few seconds longer but ensure you aren't committing stuff with errors.

---

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,9 @@
     "start": "react-scripts start",
     "build": "yarn verify"
   },
+  "pre-commit": [
+    "verify"
+  ],
```
Comment (trevordmiller) [inline package.json:23]: This new config tells the project to run the verify script before committing and if it has errors it doesn't commit without the `--no-verify` flag passed.

File: src/components/Logo/logo.svg [hunk 2]
```diff
@@ -1,58 +0,0 @@
...
-		c0.3-2.5-0.7-5.9-3.3-7.7c-1.3-0.8-2.7-1.3-4.2-1.3c-1.5,0-3,0.4-4.2,1.3c-2.7,1.8-3.6,5.1-3.3,7.7c1.1,10.1,0.6,21.1-10.9,26.7
-		l-0.1,0.1c-4.6,2.2-10.7,4.5-17.1,6.2c-12.9,3.5-27,5.3-41.8,5.3h-0.1c-5.8,0-10.9-4-12.4-9.7c-2.2-8.5-5.1-21.1-5.4-33
-		c-0.1-4.8,2.6-9.1,6.9-11.1c15.9-7.1,48-9.7,61.8-9.5c5,0.1,9.4,1.9,12.7,5.3c4.7-2.1,9.4-3.2,14-3.2c4.6,0,9.3,1.1,14,3.2
-		c2.3-2.4,6.4-5.2,12.7-5.3c8.2-0.1,43.6,1.4,61.8,9.5c4.4,1.9,7.1,6.3,6.9,11c-0.3,11.9-3.3,24.6-5.4,33
-		C87.7,220,82.6,224,76.7,224L76.7,224z M27.6,171.4c-6.7,0.1-8,5.7-8.2,6.9c-0.6,3.4-1.4,8.6-1.7,14.2c-0.1,5.4,1.9,9.2,5.8,11.1
-		c14.5,6.5,31.4,9.7,51.6,9.8c1.6,0,3-1,3.3-2.6c1.9-7.5,4.5-18.7,4.8-28.9c0-1-0.5-1.9-1.4-2.3C67.9,173.3,38.4,171.2,27.6,171.4z
-		 M-83.4,179.5c-0.9,0.4-1.5,1.3-1.4,2.3c0.3,10.2,2.9,21.4,4.8,28.9c0.4,1.5,1.8,2.6,3.3,2.6c20.2-0.1,37.1-3.3,51.7-9.8
-		c3.9-1.9,5.8-5.6,5.8-11.1c-0.2-4.3-0.8-9.3-1.7-14.2c-0.2-1.1-1.6-6.8-8.2-6.9C-35.6,171.3-67.8,172.5-83.4,179.5z"/>
-</g>
-</svg>
```
Comment (trevordmiller) [inline src/components/Logo/logo.svg:58]: Removing this for now as it was unused and also doesn't build (.svg files don't build with library currently)

File: src/utils/Card/index.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
 import React, { PropTypes } from 'react'
 import { keys } from 'lodash'
 import styled from 'styled-components'
-import { CourseMeta, CourseHeader } from './CourseCard'
-import { LessonMeta, LessonHeader } from './LessonCard'
-import { PlaylistMeta, PlaylistHeader } from './PlaylistCard'
+import { CourseMeta, CourseHeader } from '../../components/CourseCard'
+import { LessonMeta, LessonHeader } from '../../components/LessonCard'
+import { PlaylistMeta, PlaylistHeader } from '../../components/PlaylistCard'
```
Comment (trevordmiller) [inline src/utils/Card/index.js:6]: I would recommend the `Card` and `*Card` components be refactored; IMO, Card shouldn't have to import all of these pieces unless it is exporting or using them. I would recommend a single `Card` component that exports the different types so you can do `import {PlaylistCard} from 'Card'` and keep everything clean without side effects like this.

File: src/utils/Card/index.js [hunk 4]
```diff
@@ -106,7 +105,7 @@ const CardBody = ({title, instructor, type}) => {
     <div className={`${type === 'playlist' ? 'ph4' : ''}`}>
       <h3 className={titleHeadingClasses}>{title}</h3>
       <div className='mb4 flex justify-center items-center'>
-        <img src={instructorPortrait} alt='' className='w2 h2 br-pill mr3 pointer'/>
+        <img src={lessonJSON.instructor.avatar_url} alt='' className='w2 h2 br-pill mr3 pointer'/>
```
Comment (trevordmiller) [inline src/utils/Card/index.js:108]: Removing need for .jpg which wasn't building.

File: src/utils/Card/index.js (hunk 3 above)
Comment (tayiorbeii) [reply src/utils/Card/index.js:6]: Will work on this when the new Card designs are marked up

**Review by tayiorbeii — APPROVED** [review]

**Review by joelhooks — APPROVED** [review]

**Pull request merged**